lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -76,7 +76,7 @@ def self.run(params)
         end
 
         if upload_status_response.testing_uri
-          UI.message("🔗 Share this release with testers: #{upload_status_response.testing_uri}")
+          UI.message("🔗 Share this release with testers that have access: #{upload_status_response.testing_uri}")
```

Comment:
I believe you left a nit in the comments where "that"  was replaced with "who"

Reply:
Good call - done